### PR TITLE
libunwind is linked statically for C++ binaries

### PR DESCRIPTION
**Pull request by kramasamy**

### CODE REVIEW

File: 3rdparty/glog/BUILD [hunk 1]
```diff
@@ -8,11 +8,38 @@ package_version = "0.3.4"
...
+    "./configure --prefix=$$INSTALL_DIR --enable-shared=no",
+    "make install",
+    "rm -rf $$TMP_DIR",
+])
+
+LINUX_EXECUTE = "\n".join(COMMON_EXECUTE + [
+     './configure --prefix=$$INSTALL_DIR --enable-shared=no LIBS="-Wl,--rpath -Wl,$$UNWIND_DIR/lib -L$$UNWIND_DIR/lib" CPPFLAGS=-I$$UNWIND_DIR/include LDFLAGS=-L$$UNWIND_DIR/lib',
+     'make install LIBS="-Wl,--rpath -Wl,$$UNWIND_DIR/lib -L$$UNWIND_DIR/lib" CPPFLAGS=-I$$UNWIND_DIR/include LDFLAGS=-L$$UNWIND_DIR/lib',
+     'rm -rf $$TMP_DIR',
+])
```
Comment (billonahill) [inline 3rdparty/glog/BUILD]: This block above seems to be the same as that in gperftools/BUILD. Any way we can share this and not duplicate logic?

Comment (kramasamy) [inline 3rdparty/glog/BUILD]: since it is across packages - not sure how to share it in bazel.

**Comment by objmagic** [comment]
What's our plan on this issue? It seems that no one is working on it even though it's of high priority... (I am not focusing on this right now)

**Comment by kramasamy** [comment]
@objmagic - @zuyu and I are working with the travis folks.

**Comment by objmagic** [comment]
any update on this? should this be included in 0.14.5?

**Comment by objmagic** [comment]
cc @congwang who could help look into this

**Comment by objmagic** [comment]
moving to #1631 and close this.

**Pull request closed without merging**